index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="style.css">
+ <title>Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="navbar">
+      <a href="#" class="logo">
+        <h1 class="logo">My Logo</h1>
+      </a>
+      <div class="toggle">
+        <img src="/image/Union.png" alt="" class="toggle-image">
+      </div>
```

Comment:
- [ ] Its a requirement that the hamburger menu should be contained in a `nav` tag. Kindly fix this